src/Uno.UI.RuntimeTests/Tests/Windows_UI_StartScreen/Given_JumpListItem.cs

```diff
@@ -111,6 +111,29 @@ public void When_Logo_Uri_Is_ms_appx()
 			item.Logo = new Uri("ms-appx:///Test.png");
 			Assert.AreEqual("ms-appx:///Test.png", item.Logo.AbsoluteUri);
 		}
+
+		[TestMethod]
+		public async Task When_Description_Is_Empty_No_Exception()
+		{
+			try
```

Comment:
Test will need to run on the UI thread

Reply:
See this for an example:

https://github.com/unoplatform/uno/blob/77e733e95e6e198b90eb24510e0225dfc8979fea/src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_TextBlock.cs#L33-L35

Reply:
Fixed to run on UI thread